src/main/resources/native/android/c/launcher.c

```diff
@@ -208,3 +208,16 @@ void JVM_RawMonitorEnter() {
 void JVM_RawMonitorExit() {
     fprintf(stderr, "We should never reach here (JVM_RawMonitorExit)\n");
 }
+
+// AWT
```

Comment:
Add a bit more context here so that we remember later why this is here (and can remove it):
GraalVM native-image explicitly adds (unresolved) references to libawt so we need to make sure the JNI_OnLoad symbols are there.

Reply:
done

Reply:
Add the cause there (the text I have), since we need to remember where the `U` comes from so that we can remove it as soon as it is fixed in GraalVM.